de-webapp/src/main/java/org/iplantc/de/server/websocket/MessageHandler.java

```diff
@@ -46,6 +46,11 @@ public void onOpen(WebSocket webSocket) throws IOException {
         logger.debug("user name:" + username);
 
         final Channel msgChannel = notificationReceiver.createChannel();
+        if (msgChannel == null) {
+            onError(webSocket, new WebSocketProcessor.WebSocketException("AMQP channel was null", null));
+            webSocket.close();
+            return;
+        }
```

Comment:
This may not be necessary, but should `webSocket.close()` also be called if `notificationReceiver.createChannel()` does not return `null`?

Reply:
If it's possible for there to be another case where there's an issue with the channel but the channel is not null, then perhaps.  Outside of that though, I don't think the websocket needs to be closed.